### PR TITLE
docs(legal): update legal framework and policies (November 2023)

### PR DESCRIPTION
**Pull request by mffap**
## Why we are making this changes

- We reworked our **pricing model** and updated our terms and service descriptions accordingly
- Changes were made to our third party sub-processors
  - Adding Clickhouse, Inc. for data warehouse services
  - Adding Remote Europe Holding, B.V. for global payroll
  - Adding Mohlmann Solutions SRL for global payroll
- We updated our **Data Processing Agreement** and **policies** for more clarity
- Some editorial changes were necessary to bring more consistency to the legal section

### Support Framework Agreement

We made a significant change to our Agreement by combining the **Support Service Framework Agreement** with our [Framework Agreement](https://zitadel.com/docs/legal/terms-of-service). Special terms that only affect customers with a Support Service / Enterprise Agreement can be found in the newly created "Annex for ZITADEL Enterprise and Support Services". 
**No material changes to the contents of these agreements were introduced.**

### Data processing agreement

- Provided more information about how we handle security incidents
- Updated "Assistance in responding to requests" clarifying the responsibilities of each party; specifically that ZITADEL 
   - will notify the customer of requests from end-users and assists rather than directly interact with end-users
   - will notify the customer if relevant personal data has been disclosed to authorities, unless prohibited
- Clarification of "Information and control rights of the customer"
- Updated "Involvement of subcontracted processors" regarding measures and standard contractual clauses; this information was provided in the privacy policy, but is duplicated to the DPA for clarity
- Object to changes of new sub-processors in 30 days
- Reference to sub-processor list in legal section (prior: zitadel.com/trust)
-  Annex regarding security measures: password hash algorithm can new be chosen by customers during upload; added vendor management policies

### Privacy policy

- Reference to sub-processor list in legal section (prior: zitadel.com/trust)
- Sub-headings for "Disclosure to third parties"

### Other Policies

- Feature Development: The policy was moved from zitadel.com/pricing#feature-development to zitadel.com/legal/policies
- Acceptable Use Policy: Add further details and limits of Actions feature

### Service Descriptions

- Introduces a separate billing section
- Cloud Service Description: Clarification that the GDPR Safe Country region includes also EU member states
- Update to support service per subscription plans
- Official support language is English, removing German

### CODE REVIEW

File: docs/docs/legal/subprocessors.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+--- 
+
+In order to achieve the best possible transparency we publish which sub-processors and services we use to provide ZITADEL and related services.
+The table shows what activity each entity performs.
+More information about each activity is provided directly below.
+This explains the limited processing of customer data the entity is authorized to perform.
+
+We regularly audit all data processing agreements that we have with our sub-processors to guarantee that they adhere to the same level of privacy as ours to protect your personal data.
+
+The following table indicates which sub-processors have access to  end-user data. We try to minimize the number of sub-processors that handle end-user data on our behalf to reduce any vendor related risks.
```
Comment (hifabienne) [inline docs/docs/legal/subprocessors.md:14]:
```suggestion
The following table indicates which sub-processors have access to end-user data. We try to minimize the number of sub-processors that handle end-user data on our behalf to reduce any vendor related risks.
```

File: docs/docs/legal/service-description/billing.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+### Daily active user (DAU)
+
+Daily Active Users (DAU) are counted as users who authenticate or refresh their token during the given day.
+To calculate the monthly quota we take the sum of DAU over a given month.
+Included are users that either login with local accounts or users that login with an external identity provider.
+Service user who authenticate or access the management API are counted against Daily Active Users.
+
+### Active external identity providers
+
+To calculate the monthly quota we take maximum activated external identity providers on each day over a given month.
```
[the remark this answers is not in the document]
Comment (mffap) [reply docs/docs/legal/service-description/billing.md:35]: I would leave the "bucketing" of units to the pricing page.

File: docs/docs/legal/service-description/support-services.mdx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+#### Support in subscription plans
+
+Support features for ZITADEL Cloud subscriptions are as follows:
+
+Subscription Plans | Free | Production | Enterprise Cloud
+--- | --- | --- | ---
+[Support hours](#support-hours) | Business hours | Business hours | bespoke (up to 24x7)
+[Response Time](#slo---initial-response-time) (Severity 1) | Best effort | 48 business hours | bespoke (as low as 30min)
+[Community support](#community-support) | yes | yes | yes
+[Professional support](#professional-support) | no | yes | yes
```
[the remark this answers is not in the document]
Comment (mffap) [reply docs/docs/legal/service-description/support-services.mdx:36]: We include basic support with response time of 48 business hours in the Pro plan. Free plans do not get access to support. The SLA can be upgraded by booking extended support

File: docs/docs/legal/service-description/billing.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Monthly quota means the available usage per measure for one billing period.
+The quota is reset to zero with the start of a new billing period.
+
+### Daily active user (DAU)
+
+Daily Active Users (DAU) are counted as users who authenticate or refresh their token during the given day.
+To calculate the monthly quota we take the sum of DAU over a given month.
+Included are users that either login with local accounts or users that login with an external identity provider.
+Service user who authenticate or access the management API are counted against Daily Active Users.
```
[the remark this answers is not in the document]
Comment (mffap) [reply docs/docs/legal/service-description/billing.md:31]: Service accounts might require a different definition. A token will only be used when the service account uses a privately signed JWT, but not for PAT or client credentials. In these cases the account would be active, but not get a token and should be counted if it accesses the mgmt APIs to manipulate resources.

File: docs/docs/legal/annex-support-services.mdx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+--- 
+
+This annex of the [Framework Agreement](terms-of-service) describes the commercial support services (**Support Services**, **Enterprise License**, or **Enterprise Agreement**) for units of ZITADEL software products (**Unit**), if not otherwise defined a Unit refers to a is a single, dedicated setup of an application or service covered under an Enterprise agreement.
+
+The customer relationship (**Framework Agreement** or **The Agreement**) is created by the **Customer** (**"you"**) by accepting a **Purchase Order** (**"PO"**) for the specified Support Services (**Booking**). Jointly you and ZITADEL will be referred to as **the Parties**. The terms of service (**"TOS"**) outlined in this document establish the most important points of this Framework Agreement – independently of the use of any other services.
+
+### Term
+
+Coverage under this Agreement will start with Booking of Support Services, for a minimum period of 12 months.
+Support Services agreements will automatically renew for additional one year terms upon submission of a purchase order for renewal, unless either you or ZITADEL provides written notice (email sufficient) of termination of any such term.
```
Comment (fforootd) [inline docs/docs/legal/annex-support-services.mdx:14]:
```suggestion
Support Services agreements will automatically renew for additional one-year term upon submission of a purchase order for renewal, unless either you or ZITADEL provides written notice (E-Mail sufficient) of termination of any such term.
```

File: docs/docs/legal/annex-support-services.mdx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+### Disaster recovery
+
+You are solely responsible to ensure appropriate backup and disaster recovery of Units managed by you.
+Any liability for damages, indirect or direct, in case of data loss is explicitly rejected.
+
+### Entry into force
+
+This Annex shall enter into force as of November 8, 2023.
+
+Last revised: November 8, 2023
```
Comment (fforootd) [inline docs/docs/legal/annex-support-services.mdx:68]: I guess we need to adapt this to the release date.

File: docs/docs/legal/data-processing-agreement.mdx [hunk 7]
```diff
@@ -140,4 +163,4 @@ The following measures exist for regular review, assessment and evaluation of ef
 
 This agreement is valid from 15.07.2022.
 
-Last revised: June 14, 2022
+Last revised: November 9, 2022
```
Comment (fforootd) [inline docs/docs/legal/data-processing-agreement.mdx:166]: To be updated to the correct date

File: docs/docs/legal/policies/acceptable-use-policy.md [hunk 8]
```diff
@@ -17,13 +17,28 @@ You may not:
...
 
-We may suspend or terminate your usage of our Services for any violation of this Acceptable Use Policy. You will not be entitled to any Financial Credit or compensation for any interruptions caused by violation of this policy.
+We may suspend or terminate your usage of our Services for any violation of this Acceptable Use Policy.
+You will not be entitled to any Financial Credit or compensation for any interruptions caused by violation of this policy.
+
+## Entry into force
+
+This policy is valid from November 9, 2023.
+
+Last revised November 9, 2023
```
Comment (fforootd) [inline docs/docs/legal/policies/acceptable-use-policy.md:44]: Did this policy not already exist?

Would not the last revised only need to be changed?

File: docs/docs/legal/policies/privacy-policy.mdx [hunk 9]
```diff
@@ -185,4 +191,4 @@ If you have any questions about our data processing, please email us or contact
 
 This privacy policy is valid from July 15, 2022.
 
-Last revised: December 2, 2022
\ No newline at end of file
+Last revised: November 9, 2023.
```
Comment (fforootd) [inline docs/docs/legal/policies/privacy-policy.mdx:194]: To be updated

File: docs/docs/legal/service-description/billing.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
+---
+title: Pricing and billing of ZITADEL services
+sidebar_label: Billing
+custom_edit_url: null
+--- 
+
+This annex of the [Framework Agreement](../terms-of-service) describes the pricing and billing of our Services.
+
+Last revised: November 9, 2023
```
Comment (fforootd) [inline docs/docs/legal/service-description/billing.md:9]: Does this also need a "went into effect date"?

File: docs/docs/legal/service-description/billing.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+### Monthly quota
+
+Monthly quota means the available usage per measure for one billing period.
+The quota is reset to zero with the start of a new billing period.
+
+### Daily active user (DAU)
+
+Daily Active Users (DAU) are counted as users who authenticate or refresh their token during the given day.
+To calculate the monthly quota we take the sum of DAU over a given month.
```
Comment (fforootd) [inline docs/docs/legal/service-description/billing.md:29]: I am not sure about the word quota here. Would it not be better if we use "amount"

File: docs/docs/legal/service-description/billing.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+Access and runtime logs (logs) means logs that are available about your instance.
+Logs may contain information about success or failure reasons for API requests and Action executions, output from Actions, rate limit violations, and system health.
+You might be able to retrieve logs only for a limited period of time based on your subscription.
+
+### Custom domains
+
+Custom domains mean domains that you can configure to reach your ZITADEL instance.
+By default ZITADEL creates a custom domain for you when creating new instances.
+Each additional custom domain is charged.
```
Comment (fforootd) [inline docs/docs/legal/service-description/billing.md:76]:
```suggestion
Besides the included amount each additional custom domain is charged.
```

Just as a clarification

File: docs/docs/legal/service-description/cloud-service-description.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
+---
+title: Service description for ZITADEL Cloud and ZITADEL Enterprise
+sidebar_label: Service description
+custom_edit_url: null
+--- 
+
+This annex of the [Framework Agreement](../terms-of-service) describes the services offered by us.
+
+Last revised: November 9, 2023
```
Comment (fforootd) [inline docs/docs/legal/service-description/cloud-service-description.md:9]: Into force missing here?

File: docs/docs/legal/service-description/cloud-service-description.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+
+## Services offered
+
+### ZITADEL Cloud
+
+[ZITADEL Cloud](https://zitadel.com) is a fully managed cloud service of the [ZITADEL software](https://github.com/zitadel).
+
+You will benefit from the same software as the open source software, but we take care of the hosting, maintenance, backup, scaling, and operational security. The cloud service is managed and maintained by the team that also develops the software.
```
Comment (fforootd) [inline docs/docs/legal/service-description/cloud-service-description.md:17]:
```suggestion
You will benefit from the same software as the open-source project, but we take care of the hosting, maintenance, backup, scaling, and operational security. The cloud service is managed and maintained by the team that also develops the software.
```

File: docs/docs/legal/service-description/cloud-service-description.md [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+
+When creating a new instance, you are able to choose a [data location](#data-location).
+We follow a single-provider strategy by minimizing the involved [sub-processors](../subprocessors.md) to increase security, compliance, and performance of our services. [Billing](./billing.md) is based on effective usage of our services.
+
+### Enterprise license / self-hosted
+
+ZITADEL Enterprise license allows you to use the [ZITADEL software](https://github.com/zitadel) on your own data center or private cloud.
```
Comment (fforootd) [inline docs/docs/legal/service-description/cloud-service-description.md:24]:
```suggestion
The ZITADEL Enterprise license allows you to use the [ZITADEL software](https://github.com/zitadel) on your own data center or private cloud.
```

File: docs/docs/legal/service-description/service-level-description.md [hunk 16]
```diff
@@ -1,33 +1,42 @@
...
+sidebar_label: Service Level
 custom_edit_url: null
 --- 
 
-## Introduction
-
-This annex of the [Framework Agreement](terms-of-service) describes the service levels offered by us for our Services (ZITADEL Cloud).
+This annex of the [Framework Agreement](../terms-of-service) describes the service levels offered by us for our Services (ZITADEL Cloud).
 
 Last revised: June 14, 2022
```
Comment (fforootd) [inline docs/docs/legal/service-description/service-level-description.md:9]: I think this need updating too.

File: docs/docs/legal/service-description/support-services.mdx [hunk 17]
```diff
@@ -0,0 +1,254 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This annex of the [Framework Agreement](../terms-of-service) and the [Support Service Terms and Conditions](../annex-support-services) describes the support services offered by us for our Services.
+
+Support Services for products and services provided by ZITADEL is offered to customers according to the terms and conditions outlined in this document.
+The customer may purchase support services from ZITADEL (CAOS Ltd.) directly.
+
+Last revised: November 8, 2023
```
Comment (fforootd) [inline docs/docs/legal/service-description/support-services.mdx:15]: Into force needed?

File: docs/docs/legal/service-description/support-services.mdx [hunk 18]
```diff
@@ -0,0 +1,254 @@
...
+Severity 3| Best effort | 12 business hour | 12 business hours
+Severity 4| Best effort | 24 business hour | 24 business hours
+
+If we fail to provide the initial response time objective, you will be entitled to service credits. For every 15 minutes exceeding the state objective, 1 day will be added as extension to the current term.
+
+### Communication
+
+#### Community support
+
+Community support for ZITADEL is available on our website, our [public chat](https://zitadel.com/chat), and [Github](https://github.com/zitadel/).
```
Comment (fforootd) [inline docs/docs/legal/service-description/support-services.mdx:102]:
```suggestion
Community support for ZITADEL is available on our website, our [public chat](https://zitadel.com/chat), and [GitHub](https://github.com/zitadel/).
```

File: docs/docs/legal/service-description/support-services.mdx [hunk 19]
```diff
@@ -0,0 +1,254 @@
...
+We offer TAM services only bundled with specific subscription plans, and the option to add more TAM hours per period to these plans.
+If you require consulting for your projects, please request a quote via our [website](https://zitadel.com/contact).
+
+### Onboarding support
+
+Our onboarding support should help you, as a new customer, to get a better understanding on how to integrate ZITADEL into your solution, how to tackle the migration, and ensure a highly-available day-to-day operation.
+
+Onboarding support services can be offered to customers that enter a ZITADEL Cloud or a ZITADEL Enterprise subscription.
+
+If you intend to use the open source version exclusively then please join our community chat or Github.
```
Comment (fforootd) [inline docs/docs/legal/service-description/support-services.mdx:138]:
```suggestion
If you intend to use the open-source version exclusively then please join our community chat or GitHub.
```

File: docs/docs/legal/subprocessors.md [hunk 20]
```diff
@@ -0,0 +1,21 @@
...
+The table shows what activity each entity performs.
+More information about each activity is provided directly below.
+This explains the limited processing of customer data the entity is authorized to perform.
+
+We regularly audit all data processing agreements that we have with our sub-processors to guarantee that they adhere to the same level of privacy as ours to protect your personal data.
+
+The following table indicates which sub-processors have access to end-user data. We try to minimize the number of sub-processors that handle end-user data on our behalf to reduce any vendor related risks.
+Some providers are used by default, but you can opt-out of the default provide and replace the sub-processor by a provider of your choice.
+
+Last revised: November 9, 2023
```
Comment (fforootd) [inline docs/docs/legal/subprocessors.md:17]: Into force?

**Review by fforootd — COMMENTED** [review]
Wow great work 🥇 

Left some small comments besides that it looks good

**Comment by mffap** [comment]
> Wow great work 🥇
> 
> Left some small comments besides that it looks good

@fforootd I've now opted to leave out the effective since / entry into force date and instead consistently place on the very top "Last updated on ..." for each file. 

All creation dates and updates can anyways be audited in the commit history: 
https://github.com/zitadel/zitadel/commits/main/docs/docs/legal

**Comment by fforootd** [comment]
> > Wow great work 🥇
> > Left some small comments besides that it looks good
> 
> @fforootd I've now opted to leave out the effective since / entry into force date and instead consistently place on the very top "Last updated on ..." for each file.
> 
> All creation dates and updates can anyways be audited in the commit history: https://github.com/zitadel/zitadel/commits/main/docs/docs/legal

Understood, that makes sense.

**Review by fforootd — APPROVED** [review]

**Pull request merged**